### PR TITLE
Adding exporter for Citrix ADC (formerly Citrix NetScaler)

### PR DESCRIPTION
**Pull request by bhardwajk**
Signed-off-by: Komal Bhardwaj <komal.bhardwaj@citrix.com>

### CODE REVIEW

**Comment by roidelapluie** [comment]
Please DCO upon modifying the file, not on the pull request:

soff

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -116,6 +117,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Tinyproxy exporter](https://github.com/igzivkov/tinyproxy_exporter)
    * [Varnish exporter](https://github.com/jonnenauha/prometheus_varnish_exporter)
    * [WebDriver exporter](https://github.com/mattbostock/webdriver_exporter)
+
```
Comment (SuperQ) [inline content/docs/instrumenting/exporters.md:120]: Spurious newline.

**Comment by brian-brazil** [comment]
Thanks for sharing, I'm a bit confused by this exporter. Where is the actual source code? It doesn't seem to be in the linked repo.

Can you explain why the readme mentions Zipkin, Elasticsearch and Kafka? Why do the deployment docs mention the pushgateway?  Can you explain what sort of metrics this provides, and if what it covers is more/less than what's available from SNMP?

**Comment by roidelapluie** [comment]
At $customer I am using https://github.com/rokett/Citrix-NetScaler-Exporter. Did you compare the metrics of both? Did you engage with the community?

**Comment by bhardwajk** [comment]
> Thanks for sharing, I'm a bit confused by this exporter. Where is the actual source code? It doesn't seem to be in the linked repo.
> 
> Can you explain why the readme mentions Zipkin, Elasticsearch and Kafka? Why do the deployment docs mention the pushgateway? Can you explain what sort of metrics this provides, and if what it covers is more/less than what's available from SNMP?

Hi @brian-brazil , this exporter is not open source. Also COE exports to multiple endpoints including Zipkin, Elasticsearch, Kafka, Prometheus. It can export HTTP, TCP related metrics among others.

**Comment by bhardwajk** [comment]
> At $customer I am using https://github.com/rokett/Citrix-NetScaler-Exporter. Did you compare the metrics of both? Did you engage with the community?

https://github.com/citrix/citrix-observability-exporter/ is officially supported by Citrix Inc for exporting metrics from Citrix ADC

**Comment by roidelapluie** [comment]
> > At $customer I am using https://github.com/rokett/Citrix-NetScaler-Exporter. Did you compare the metrics of both? Did you engage with the community?
> 
> https://github.com/citrix/citrix-observability-exporter/ is officially supported by Citrix Inc for exporting metrics from Citrix ADC

There is a preference for official exporters, we do need to check that it is a reasonable replacement though. Engaging with the existing exporters is encouraged.

**Comment by bhardwajk** [comment]
> Please DCO upon modifying the file, not on the pull request:
> 
> soff

Closing this PR and updated DCO in original file modification in a new PR https://github.com/prometheus/docs/pull/1669

**Pull request closed without merging**